tests/framework/common_apps.go

```diff
@@ -472,6 +473,30 @@ func (td *OsmTestData) SimpleDeploymentApp(def SimpleDeploymentAppDef) (corev1.S
 	return serviceAccountDefinition, deploymentDefinition, serviceDefinition, nil
 }
 
+// GetOSSpecificSleepPod returns a simple OS specific busy loop pod.
+func (td *OsmTestData) GetOSSpecificSleepPod(sourceNs string) (corev1.ServiceAccount, corev1.Pod, corev1.Service, error) {
+	if td.ClusterOS == constants.OSWindows {
+		return Td.SimplePodApp(SimplePodAppDef{
+			Name:      "client",
+			Namespace: sourceNs,
+			Command:   []string{"cmd", "/c"},
+			Args:      []string{"FOR /L %N IN () DO ping -n 30 127.0.0.1> nul"},
+			Image:     WindowsNanoserverDockerImage,
```

Comment:
I love that we can use nanoserver here!

Reply:
It takes some hacking but I really want to have all the images be nano based because it makes things so much easier